src/components/portrait/portrait.component.js

```diff
@@ -30,6 +30,7 @@ class Portrait extends React.Component {
       alt,
       darkBackground,
       gravatar,
+      icon,
```

Comment:
I'd recommend changing the name of this prop to `iconType`. This will keep it inline with the rest of our components like Search, Accordion, Confirm and Button for example.